client-app/components/molecules/textarea/Textarea.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="relative">
+    <textarea
+      v-model="text"
+      :rows="rows"
+      :maxlength="maxLength"
+      class="border rounded-sm w-full p-3"
+      :class="resizeClass"
+    ></textarea>
+    <p class="absolute text-xs text-gray-300 bottom-5 right-3">{{ symbolsCount }}</p>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed, toRef } from "vue";
+const props = defineProps({
+  rows: {
+    type: Number,
+    required: true,
+  },
+
+  maxLength: {
+    type: Number,
+    required: true,
```

Comment:
This is optional, not every textarea will have such behavior, not needed to force this prop as required.